lightning/src/ln/msgs.rs

```diff
@@ -75,6 +75,8 @@ pub enum DecodeError {
 pub struct Init {
 	/// The relevant features which the sender supports
 	pub features: InitFeatures,
+	/// The network address which the sender provides
```

Comment:
This should describe why we're passing this, and who's address is described (the recipient, not the sender).